### ISSUE TITLE
Accessibility: <label for=...> missing

### ISSUE THREAD

**Issue by trob** [Fabrik/fabrik#453]
In fabrik2 forms were using
`<label for="element"...`
as it should be for accessibility reasons (it's using `<label for=...>`  in details view also, which shouldn't be there without `<input..>`, but ok, it's fabrik2)

In fabrik3 my accessibility checker is throwing a bunch of warnings because of missing labels
